docs/source/gpu.rst

```diff
@@ -1,22 +1,43 @@
 GPUs
 ====
 
-Dask works with GPUs in a few ways.
-
-
-Custom Computations
--------------------
-
-Many people use Dask alongside GPU-accelerated libraries like PyTorch and
-TensorFlow to manage workloads across several machines.  They typically use
-Dask's custom APIs, notably :doc:`Delayed <delayed>` and :doc:`Futures
-<futures>`.
-
-Dask doesn't need to know that these functions use GPUs.  It just runs Python
-functions.  Whether or not those Python functions use a GPU is orthogonal to
-Dask.  It will work regardless.
-
-As a worked example, you may want to view this talk:
+Graphics Processing Units (GPUs) can dramatically speed up certain numerical computations like
+machine learning model training. Tools like RAPIDS, TensorFlow, and PyTorch can be run on
+GPUs instead of a standard CPU to produce results an order of magnitude faster (or more).
+Dask connects nicely with these different tools--while the particular hardware might be different
+when using GPUs, so long as the Python code can run on a set of machines
+Dask can coordinate running that code.
+
+There are many different ways to use Dask and GPUs--from training one large machine learning model
+across many workers concurrently, to doing hyperparameter tuning where each worker has a different
+set of hyperparameters to test. This page will walk through a few of them.
+
+GPU-accelerated libraries for large computations
+------------------------------------------------
+
+Tools like PyTorch and TensorFlow both have the capability to use GPUs as well as the ability to
+train a single model across many machines. This has very practical applications: large models like
+`GPT-3 <https://arxiv.org/abs/2005.14165>`_ are trained on clusters of computers in this manner.
+Dask can be used as a straightforward manner of coordinating such a cluster to train a model. while
+there are benefits to using Dask for training very large models, there may still be speed
+improvements for training models with fewer parameters as well.
+
+.. image:: images/gpu-1-libraries.png
+   :alt: Training a model over GPUs across multiple workers
+   :width: 80%
+
+To take one example, PyTorch has the
+`DistributedDataParallel <https://pytorch.org/tutorials/intermediate/ddp_tutorial.html>`_ (DDP)
```

Comment:
It seems odd to call out this one technology.  In my experience PyTorch-DDP is not the most prominent distributed training system (Horovod probably takes that claim?).  If you want to include DDP (for example because you're hoping that people then search for dask pytorch ddp and find your software or materials) then I recommend putting it in a list of software projects that are as-or-more prominent.